rollup.config.js

```diff
@@ -61,7 +42,6 @@ export default {
 		nodeResolve({
 			jsnext: true,
 			module: true,
-			skip: ['react', 'preact'],
```

Comment:
I think `skip` is deprecated with latest version of plugin and `external` is preffered